### PR TITLE
Library Genesis: Sorting results by preferred file formats

### PR DESCRIPTION
**Pull request by yz3440**
## Description

Users can sort the search result based on the preferred file formats (pdf, djvu, etc.).

## Screencast

screencast

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/library-genesis/README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <p align="center">
+  <img src="assets/command-icon.png" height="128">
```
Comment (danielsequeira) [inline extensions/library-genesis/README.md:2]: @yz3440 this unfortunately won't work atm, we have a bug that doesn't let images using their relative path in html code render. You need to use the absolute path from github or markdown lang to render the image

File: extensions/library-genesis/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Library Genesis Changelog
 
+## [Added sort by file formats] - 2022-07-21
+
+- User can sort the search result based on the preferred file formats.
```
Comment (danielsequeira) [inline extensions/library-genesis/CHANGELOG.md:5]:
```suggestion
User can sort the search result based on the preferred file formats.
```

File: extensions/library-genesis/package.json [hunk 3]
```diff
@@ -80,22 +80,45 @@
       "default": "0",
       "required": true
     },
+    {
+      "name": "searchPriority",
+      "title": "Priotitized Search By",
```
Comment (danielsequeira) [inline extensions/library-genesis/package.json:85]:
```suggestion
      "title": "Priotitize Search By",
```

File: extensions/library-genesis/README.md (hunk 1 above)
Comment (yz3440) [reply extensions/library-genesis/README.md:2]: Thanks for letting me know!

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**